Apps/Sandcastle/gallery/Ordering Geometry.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
```

Comment:
I would rename this to `Z-Indexing Geometry`, to both be consistent with our internal terminology and because Z-Indexing is the standard term.